src/Service/Startup.cs

```diff
@@ -892,9 +905,32 @@ private void ConfigureAzureLogAnalytics(RuntimeConfig runtimeConfig)
                     return;
                 }
 
+                if (string.IsNullOrEmpty(AzureLogAnalyticsOptions.Auth?.CustomTableName))
+                {
+                    _logger.LogInformation("Logs won't be sent to Azure Log Analytics because the Custom Table Name is not available in the config file.");
+                    return;
+                }
+
+                if (string.IsNullOrEmpty(AzureLogAnalyticsOptions.Auth?.DcrImmutableId))
+                {
+                    _logger.LogInformation("Logs won't be sent to Azure Log Analytics because the DCR Immutable Id is not available in the config file.");
+                    return;
+                }
+
+                if (string.IsNullOrEmpty(AzureLogAnalyticsOptions.Auth?.DceEndpoint))
+                {
+                    _logger.LogInformation("Logs won't be sent to Azure Log Analytics because the DCE Endpoint is not available in the config file.");
+                    return;
```

Comment:
If its only logging as an information, why are we returning pre-maturely without ?

I don't think dab should die if we Log Analytics is incorrectly configured. You can check with @JerryNixon to confirm this.